src/tests/i18n.spec.js

```diff
@@ -0,0 +1,133 @@
+            },
+            getIn: function (data) {
+              if (data[0] === 0) {
+                return '#333';
+              }
+              return '#444';
+            },
+          };
+        },
+        onMonthChange: function () {},
+        onYearChange: function () {},
+        bemBlock: 'DateRangePicker',
+        locale: props.locale || 'en',
+      }, props);
+
+
+      return (<CalendarMonth {...props} />);
+    };
+
+    this.useShallowRenderer = (props) => {
+      this.shallowRenderer = TestUtils.createRenderer();
+      this.shallowRenderer.render(getCalendarMonth(props));
+      this.renderedComponent = this.shallowRenderer.getRenderOutput();
+      this.container = this.renderedComponent.props.children[0];
+      this.table = this.renderedComponent.props.children[1];
+    };
+
+    this.useDocumentRenderer = (props) => {
+      this.component = this.renderedComponent = TestUtils.renderIntoDocument(getCalendarMonth(props));
+    };
+
+    this.firstOfMonth = moment();
+  });
+
+  afterEach( function () {
+    if (this.component) {
+      React.unmountComponentAtNode(React.findDOMNode(this.component).parentNode);
+    }
+  });
+
+  it('renders the proper month header', function () {
+
+    require(`moment/locale/${testLocale}`);
+    moment.locale(testLocale);
+    this.useShallowRenderer({
+      locale: testLocale,
+    });
+
+    const currentMonth = moment().format('MMMM');
+    const headerMonthLabel = this.container.props.children[0].props.children[0];
+
+    expect(headerMonthLabel).toEqual(currentMonth);
+  });
+
+  it('renders the proper month options', function () {
+
+    require(`moment/locale/${testLocale}`);
+    moment.locale(testLocale);
+    this.useShallowRenderer({
+      locale: testLocale,
+    });
+
+    const months = moment.localeData(testLocale)._months;
+
+    const headerMonthSelect = this.container.props.children[0].props.children[1];
+
+    headerMonthSelect.props.children.map((option, index) => {
+      const optionText = option.props.children;
+      expect(optionText).toEqual(months[index]);
+    });
+
+  });
+
+  it('renders the proper year header', function() {
+    require(`moment/locale/${testLocale}`);
+    moment.locale(testLocale);
+    this.useShallowRenderer({
+      locale: testLocale,
+    });
+
+    const currentYear = moment().format('YYYY');
+    const headerYearLabel = this.container.props.children[2].props.children[0];
+
+    expect(headerYearLabel).toEqual(currentYear);
+
+  });
+
+  it('renders the proper year options', function () {
+
+    const years = _.map(_.range(0, 4), (val) => {
+      return moment().add(val, 'y').format('YYYY');
+    });
+
+    require(`moment/locale/${testLocale}`);
+    moment.locale(testLocale);
+    this.useShallowRenderer({
+      locale: testLocale,
+    });
+
+    const headerYearSelect = this.container.props.children[2].props.children[1];
+
+    _.map(_.compact(headerYearSelect.props.children), (option, index) => {
```

Comment:
@thaerlabs Could we change this to `each` - as we simply want to loop over every value for test purposes

Also, should we add an assertion here that there are actually any children present?